readme.md

```diff
@@ -60,6 +60,7 @@ $ np --help
     --no-yarn           Don't use Yarn
     --contents          Subdirectory to publish
     --no-release-draft  Skips opening a GitHub release draft
+    --test-script       User defined test to run before publish
```

Comment:
```suggestion
    --test-script       User-defined test to run before publish
```

And it's not running a "test"

It's also not clear that this replaces the normal test script.